mmcv/ops/csrc/pytorch/all_pairs_correlation.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) OpenMMLab. All rights reserved.
+#include <iostream>
+#include "pytorch_cuda_helper.hpp"
```

Comment:
including "pytorch_cuda_helper.hpp"  duplicately caused the  bug
delete ``#include "pytorch_cuda_helper.hpp"``